src/main/java/org/pegdown/ToHtmlSerializer.java

```diff
@@ -49,7 +49,7 @@ public ToHtmlSerializer(LinkRenderer linkRenderer) {
     }
 
     public ToHtmlSerializer(LinkRenderer linkRenderer, List<ToHtmlSerializerPlugin> plugins) {
-    	this (linkRenderer, Collections.<String, VerbatimSerializer>emptyMap(), plugins);
```

Comment:
Nice formatting catch!
One reason why I have switched to auto-formatting for all current projects in the mean-time...